src/App.js

```diff
@@ -1,12 +1,86 @@
-import logo from './logo.svg';
 import './App.css';
+import { getDatabase, ref, onValue, push, remove } from 'firebase/database';
+import firebase from './firebase';
+import { useState, useEffect } from 'react';
+import Header from './Header';
+import Form from './Form';
+import Results from './Results';
+import Footer from './Footer';
 
 function App() {
+
+  const initialValues = {
```

Comment:
so i noticed that once i click on a color input, i can not customize it right away and click okay, it does not really pick up anything. I feel like this is a color picker thing but just wanted to let you know in case you wanna make that clear for the user. I can explain myself more if this comment is a bit vague.